src/label_studio_sdk/label_interface/control_tags.py

```diff
@@ -462,6 +462,23 @@ def label(
             )
         else:
             return self._label_simple(to_name=to_name, *args, **kwargs)
+        
+    def get_labels(self, regions: List[Dict]):
+        """
+        Returns the simplified representation of the label. Sort of a reverse to label() method to retrieve an input `label` from an output regions
```

Comment:
can you put an example in this docstring? It's hard to picture what the output looks like

Reply:
or add a test

Reply:
sure will do both

Reply:
done